CompuCell3D/core/pyinterface/CompuCellPython/CompuCell.i

```diff
@@ -250,7 +262,7 @@ using namespace CompuCell3D;
 %include "Logger/CC3DLogger.h"
 %include "Field3D/Point3D.h"
 %include "Field3D/Dim3D.h"
-
+%include "CompuCell3D/CC3DExceptions.h"

```

Comment:
This isn't adding CC3DException to the CompuCell python module for me on windows, resulting in failed simulation execution in at least Jupyter.

Reply:
We need to find a way to intercept c++ exceptions in Python/SWIG. This is the reason I added this code. If you have a good way that works, let me know

Reply:
We could always forward exceptions through CPython (e.g., `PyErr_SetString`)

Reply:
I would leave it the way it is now. It seems to be working fine. We can tweak it in the next versions

Reply:
and what is the error you are getting? it is "This isn't adding CC3DException to the CompuCell"? can you send me the content of the compucell.py via email? also, are you compiling or using package? I am wondering if you can try conda package. do we have jupyter instructions?

Reply:
This was for building locally, which I resolved. It was an issue with changing python versions, I think. I've also checked that this issue isn't present in the windows conda package. I'll test Jupyter for windows today.

Reply:
great, let me know how it goes.

Reply:
> do we have jupyter instructions?

Jupyter execution is done in the typical way. Since the `notebook` package is a dependency, just activate the environment after cc3d installation and then run `jupyter notebook`.

Reply:
Fixed in https://github.com/CompuCell3D/CompuCell3D/pull/631. Once this is resolved, I think this PR is good to merge.

Reply:
I merged your PR into this branch. Once you approve this PR we can move on